src/pages/steps/ContactInfoStep.tsx

```diff
@@ -0,0 +1,258 @@
+    setContactInfoState(newContactInfo);
+  };
+
+  const handleDeleteContactInfo = async (index: number) => {
+    const newContactInfo = [...contactInfoState];
+    newContactInfo.splice(index, 1);
+
+    setContactInfoState(newContactInfo);
+
+    if (isOrganizer) {
+      handleAddOrganizerContactInfo(newContactInfo);
+      return;
+    }
+
+    await handleAddContactInfoMutation(newContactInfo);
+  };
+
+  const handleChangeContactInfoType = async (
+    event: ChangeEvent<HTMLSelectElement>,
+    index: number,
+  ) => {
+    const newType = event.target.value;
+    const newContactInfo = [...contactInfoState];
+    newContactInfo[index].type = newType;
+
+    setContactInfoState(newContactInfo);
+
+    if (newContactInfo[index].value !== '') {
+      await handleAddContactInfoMutation(newContactInfo);
+    }
+  };
+
+  return (
+    <Stack maxWidth="40rem" {...getStackProps(props)} spacing={3}>
+      {(contactInfoState ?? []).map((info, index) => {
+        return (
+          <Inline key={index} spacing={3}>
+            <Select
+              width="30%"
+              onChange={(e) => handleChangeContactInfoType(e, index)}
+            >
+              {Object.values(ContactInfoTypes).map((type) => (
+                <option value={type} selected={info.type === type} key={type}>
+                  {t(`create.additionalInformation.contact_info.${type}`)}
+                </option>
+              ))}
+            </Select>
+            <FormElement
+              width="55%"
+              Component={
+                <Input
+                  value={info.value}
+                  onChange={(e) => {
+                    const newContactInfoState = [...contactInfoState];
+                    newContactInfoState[index].value = e.target.value;
+                    setContactInfoState(newContactInfoState);
+                    setIsFieldFocused(true);
+                  }}
+                  onBlur={(e) => {
+                    setIsFieldFocused(false);
+                    handleChangeValue(e, index);
+                  }}
+                />
+              }
+              id="contact-info-value"
+              error={
+                !isFieldFocused &&
+                !isValidInfo(info.type, info.value) &&
+                t(
+                  `create.additionalInformation.contact_info.${info.type}_error`,
+                )
+              }
+            />
+
+            <Button
+              onClick={() => handleDeleteContactInfo(index)}
+              variant={ButtonVariants.DANGER}
+              iconName={Icons.TRASH}
+            ></Button>
+          </Inline>
+        );
+      })}
+      <Inline>
+        <Button
+          onClick={handleAddNewContactInfo}
+          variant={ButtonVariants.SECONDARY}
+        >
+          {contactInfoState.length === 0
+            ? t('create.additionalInformation.contact_info.add_more_singular')
+            : t('create.additionalInformation.contact_info.add_more_multiple')}
```

Comment:
You can use the built in pluralisation from i18next
https://www.i18next.com/translation-function/plurals

Look at translation `selectionTable.rowsSelectedCount` as an example

Reply:
@simon-debruijn it's not really the same case.
When there is no contact info yet I use the string `Contactgegevens toevoegen`.
When there is already contact info I use the string `Meer contactgegevens toevoegen`.